sources/models/src/aws-ecs-1-nvidia/defaults.d/51-docker-services.toml

```diff
@@ -0,0 +1 @@
+../../../shared-defaults/docker-services.toml
```

Comment:
This file has four parts:
1. docker service definition + restart command
2. docker daemon config file
3. container registry mirrors metadata
4. container registry credentials metadata

We end up overriding (2) in 53-docker-daemon.toml, and (4) in 52-aws-ecs-1.toml.

It works in the sense that we're ending up with the right values, but I wonder if there's a way to refactor this that's easier to follow.

If nothing comes to mind, I'm OK with this going in. However, it's close to the ceiling of acceptable complexity because it makes migrations in this area even harder to reason about than usual.

Reply:
I'll try to propose something to safely improve these configuration files :+1: